test/create_parameters.jl

```diff
@@ -85,4 +86,36 @@ function create_kid_parameters(toml_dict)
     return kid_params
 end
 Base.broadcastable(x::KID.Parameters.KinematicDriverParameters) = Ref(x)
+
+function create_cloudy_parameters(NM, FT)
```

Comment:
Could you add some docstring explaining what is happening here? Or link to Cloudy documentation if it's explained there? I'm asking about `NM / 3`, `1e6` and `1e-9` in `norms`, `5e0` in `kernel_func`, `5e-10` `1e-6` in `KernelTensors` and `CoalescenceData` and `vel`, etc.